accounts/keystore/file_cache.go

```diff
@@ -95,7 +95,8 @@ func skipKeyFile(fi os.FileInfo) bool {
 		return true
 	}
 	// Skip misc special files, directories (yes, symlinks too).
-	if fi.IsDir() || fi.Mode()&os.ModeType != 0 {
+	excludedModes := os.ModeDir | os.ModeNamedPipe | os.ModeSocket | os.ModeDevice
```

Comment:
Hmm, the comment seems to suggest that `symlinks` were explicitly skipped for some reason. I'd be really curious as to the decision behind that. Ping @fjl @holiman

Reply:
Here's the commit where the comment originated from: https://github.com/ethereum/go-ethereum/commit/a9f26dcd0d14c0cb9f309ebccf81e8f741fc4636#diff-f720e707cc48373f4c544a60a835446dR264

One unexpected behavior I found: when a user changes her password, geth overwrites the symlink instead of overwriting the source file. There's a few ways to solve this hitch though!